qcodes/instrument_drivers/ZI/ZIUHFLI.py

```diff
@@ -1648,65 +1689,72 @@ def _get_demod_sample(self, number: int, demod_param: str) -> float:
         datadict['phi'] = np.angle(datadict['x'] + 1j * datadict['y'], deg=True)
         return datadict[demod_param]
 
-    def _sigout_setter(self, number, mode, setting, value):
+    def _sigout_setter(self, number: int,
+                       mode: bool,
```

Comment:
sorry for being so picky, but this confusion with `mode` argument is not only in `_sigout_getter` but also here, and i guess somehwere else in the driver. would you mind fixing that as the last thing for this PR?

Reply:
Sure, this I'll hunt it down. This was a copy/paste from PR-978 and I guess I did not review much.

Reply:
done,